### ISSUE TITLE
Dates in Public Site Need to Read M-DD-YY (3/2/2018), no zeros before a #

### ISSUE THREAD

**Issue by urentmellc** [pixelphpdev/urentme#10]

**Comment by Jayantapixel** [comment]
Date format change may effect in different areas where the date has been used. So it will take time to do in all places.

**Comment by urentmellc** [comment]
this needs to be done

**Comment by pxlsz** [comment]
We have done this in following pages 'home','Item details','Booking page', 'Payment page'. Let us know if have any pages needs to be done.

**Comment by urentmellc** [comment]
will have to take time to check this, will not close the issue until its been verified...

**Comment by Jayantapixel** [comment]
Please check when you get time and close if you satisfied.

**Comment by urentmellc** [comment]
There are still issues with date formats, take the time to review all pages in the admin panel to ensure the date is in the proper format. This needs to be resolved please today.

**Comment by urentmellc** [comment]
![dateissueadminpanel](https://user-images.githubusercontent.com/34143768/37377264-13facfe4-26e6-11e8-9c1e-5d2c35ebbf70.jpeg)

**Comment by pxlsz** [comment]
Date format updated all places except 2 pages i.e. makes and models. 

We are unable to upload 2 updated files through FTP. Its showing access denied. Even unable to change file permission in server. Please talk to your server provider. Both files as attached and uplod in following path:
/public_html/app/View/Makes/admin_index.ctp
/public_html/app/View/Models/admin_index.ctp

I will send files through email since file names are same.

**Comment by pxlsz** [comment]
Files attached "urentme-14-03-2018.zip
[urentme-14-03-2018.zip](https://github.com/pixelphpdev/urentme/files/1811459/urentme-14-03-2018.zip)
".

**Comment by urentmellc** [comment]
You have my login to my server... no?

**Comment by urentmellc** [comment]
cpanel

**Comment by pxlsz** [comment]
Yes, We have logged in cpanel. We have tried to upload 2 files from cpanel but not uploading . Showing same error 'access denied'.

**Comment by urentmellc** [comment]
please retry... thank you.

**Comment by urentmellc** [comment]
has this been addressed?

**Comment by urentmellc** [comment]
Ok what needs to be done to fix this please explain... thank you. 
		
		

		Get Outlook for iOS
	




On Tue, Apr 10, 2018 at 11:18 PM -0700, "pixelphpdev" <notifications@github.com> wrote:












No, We are facing same problem. We have tried to upload make and model file but is not uploading.
